nav2_loopback_sim/nav2_loopback_sim/utils.py

```diff
@@ -63,3 +65,21 @@ def matrixToTransform(matrix):
     transform.rotation.z = quaternion[2]
     transform.rotation.w = quaternion[3]
     return transform
+
+
+def get_world2map_coordinates(world_x, world_y, map_msg):
```

Comment:
Use the bound checks in https://github.com/ros-navigation/navigation2/blob/3f311124e356dbfa33f35e23caa5474a57f99ab7/nav2_costmap_2d/src/costmap_2d.cpp#L284-L297

Reply:
Updated